Week 4/Contest (Apr 3)/Hot Hike/elijah.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+partial class Program
+{
+  static void Main()
+  {
+    int n = int.Parse(Console.ReadLine());
+    var forecast = Console.ReadLine().Split(' ').Select(s => int.Parse(s)).ToArray();
+
+    int smallestTemp = -1;
```

Comment:
I'd be careful about setting the smallestTemp to -1 for this problem. The temps can be as low as -20.

Reply:
Oof, noted. Should've looked at the constraints a little closer, fortunately it at least passed the test cases.